test/ServerRequestFactoryTest.php

```diff
@@ -427,10 +427,10 @@ public function testFromGlobalsUsesCookieHeaderInsteadOfCookieSuperGlobal()
      */
     public function testCreateFromGlobalsShouldPreserveKeysWhenCreatedWithAZeroValue()
     {
-        $_SERVER['Accept'] = '0';
+        $_SERVER['HTTP_ACCEPT'] = '0';
 
         $request = ServerRequestFactory::fromGlobals();
-        $this->assertSame('0', $request->getHeaderLine('Accept'));
+        $this->assertSame('0', $request->getHeaderLine('ACCEPT'));
```

Comment:
Please add a test scenario, instead of replacing an existing one.

Reply:
The old test is not valid, because the bugfix was spurious, but the function name takes the name that needs to be used.   This could be a revert and then I could resubmit, but I don't think that's really necessary; do you?